### ISSUE TITLE
Importing from Postgres

### ISSUE THREAD

**Issue by stevelacey** [dimitri/pgloader#630]
Would be great to support loading Postgres... from Postgres.

I am trying to grab a copy of an RDS database via an ElasticBeanstalk image for local dev, and every machine I just mentioned has a different version of Postgres – so `pg_dump` is a headache... RDS snapshots are a headache... urgh.

```
pgloader postgres://rds:5433/ebdb postgres://local:5432/mydb
```

```
FATAL An unhandled error condition has been signalled: At

  pgsql
  ^ (Line 1, Column 0, Position 0)

In context CLI-TYPE:

While parsing CLI-TYPE. Expected:

     the string "copy"
  or the string "csv"
  or the string "db3"
  or the string "dbf"
  or the string "fixed"
  or the string "ixf"
  or the string "mssql"
  or the string "mysql"
  or the string "sqlite"
  or the string "sqlite3"
```

**Comment by dimitri** [comment]
It seems to me that your local dev setup should be using the same PostgreSQL version as your RDS production database, to boot. Then `pg_dump` and `pg_restore` are really the canonical tools to use for your needs, and I would advice that you restore from a backup copy of your production database, allowing you to validate that your backups are actually dependable.

Still, in some cases pgloader support for PostgreSQL as a source would be a nice to have. Adding as a wish list item.

**Comment by dimitri** [comment]
Considering contributing to pgloader as Patron to make happen PostgreSQL support as a source for data for pgloader.

**Comment by dimitri** [comment]
It's been done and has been released in 3.6.1, by the way. See https://pgloader.readthedocs.io/en/latest/ref/pgsql.html.
